src/communication/network_interface_cl.c

```diff
@@ -7452,6 +7452,51 @@ serial_decache (OID * oid)
 #endif /* !CS_MODE */
 }
 
+/*
+ * synonym_remove_xasl_by_oid -
+ *
+ * return: NO_ERROR or error status
+ *
+ *   oid(in):
+ *
+ * NOTE:
+ */
+int
+synonym_remove_xasl_by_oid (OID * oid)
+{
+#if defined(CS_MODE)
+  int req_error;
+  OR_ALIGNED_BUF (OR_OID_SIZE) a_request;
+  OR_ALIGNED_BUF (OR_INT_SIZE) a_reply;	/* need dummy reply message */
+  char *request;
+  char *reply;
+  int status;
+
+  request = OR_ALIGNED_BUF_START (a_request);
+  reply = OR_ALIGNED_BUF_START (a_reply);
+
+  or_pack_oid (request, oid);
+
+  req_error =
+    net_client_request (NET_SERVER_SYNONYM_REMOVE_XASL_BY_OID, request, OR_ALIGNED_BUF_SIZE (a_request), reply,
+			OR_ALIGNED_BUF_SIZE (a_reply), NULL, 0, NULL, 0);
+  if (!req_error)
+    {
+      or_unpack_int (reply, &status);
```

Comment:
This line seems redundant because status is not used below.

Reply:
The serial_decache function is also related, so it seems to be solved together.
If the execution method of synonyms is changed, it can be removed, so I will not change it.